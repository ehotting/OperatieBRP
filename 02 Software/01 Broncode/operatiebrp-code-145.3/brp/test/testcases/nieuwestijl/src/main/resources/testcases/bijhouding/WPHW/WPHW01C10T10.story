Meta:
@status                 Bug
@usecase                UCS-BY.HG

Narrative: R2653 Datum aanvang geldigheid moet op of na de aanvang van de relatie liggen

Scenario:   Datum aanvang geldigheid van de Actie ligt voor de aanvang relatie
            LT: WPHW01C10T10

Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL-WPHW/WPHW01C10T10-001.xls
Given enkel initiele vulling uit bestand /LO3PL-WPHW/WPHW01C10T10-002.xls

!-- Voltrekking van een huwelijk
When voer een bijhouding uit WPHW01C10T10a.xml namens partij 'Gemeente BRP 1'

Then heeft het antwoordbericht verwerking Geslaagd

Then is in de database de persoon met bsn 899892681 wel als PARTNER betrokken bij een HUWELIJK
Then is in de database de persoon met bsn 953148233 wel als PARTNER betrokken bij een HUWELIJK

Given pas laatste relatie van soort 1 aan tussen persoon 899892681 en persoon 953148233 met relatie id 50000001 en betrokkenheid id 50000001
Given pas laatste relatie van soort 1 aan tussen persoon 953148233 en persoon 899892681 met relatie id 50000002 en betrokkenheid id 50000002

!-- Wijziging partnergegevens huwelijk
When voer een bijhouding uit WPHW01C10T10b.xml namens partij 'Gemeente BRP 1'

Then heeft het antwoordbericht verwerking Foutief

Then is het antwoordbericht gelijk aan /testcases/bijhouding/WPHW/expected/WPHW01C10T10.xml voor expressie //brp:bhg_hgpActualiseerHuwelijkGeregistreerdPartnerschap_R
