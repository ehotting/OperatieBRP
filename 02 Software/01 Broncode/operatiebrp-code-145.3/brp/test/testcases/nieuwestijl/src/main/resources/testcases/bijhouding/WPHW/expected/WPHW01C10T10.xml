<brp:bhg_hgpActualiseerHuwelijkGeregistreerdPartnerschap_R xmlns:brp="http://www.bzk.nl/brp/brp0200">
    <brp:stuurgegevens>
        <brp:zendendePartij>199901</brp:zendendePartij>
        <brp:zendendeSysteem>BRP</brp:zendendeSysteem>
        <brp:referentienummer>*</brp:referentienummer>
        <brp:crossReferentienummer>88409eeb-1aa5-43fc-8614-43055123a165</brp:crossReferentienummer>
        <brp:tijdstipVerzending>*</brp:tijdstipVerzending>
    </brp:stuurgegevens>
    <brp:resultaat>
        <brp:verwerking>Foutief</brp:verwerking>
        <brp:hoogsteMeldingsniveau>Fout</brp:hoogsteMeldingsniveau>
    </brp:resultaat>
    <brp:meldingen>
        <brp:melding brp:objecttype="Melding" brp:referentieID="comm_actie_1">
            <brp:regelCode>R2653</brp:regelCode>
            <brp:soortNaam>Fout</brp:soortNaam>
            <brp:melding>*</brp:melding>
        </brp:melding>
        <brp:melding brp:objecttype="Melding" brp:referentieID="comm_actie_2">
            <brp:regelCode>R2653</brp:regelCode>
            <brp:soortNaam>Fout</brp:soortNaam>
            <brp:melding>*</brp:melding>
        </brp:melding>
        <brp:melding brp:objecttype="Melding" brp:referentieID="comm_actie_4">
            <brp:regelCode>R2653</brp:regelCode>
            <brp:soortNaam>Fout</brp:soortNaam>
            <brp:melding>*</brp:melding>
        </brp:melding>
        <brp:melding brp:objecttype="Melding" brp:referentieID="comm_ah">
            <brp:regelCode>R2493</brp:regelCode>
            <brp:soortNaam>Deblokkeerbaar</brp:soortNaam>
            <brp:melding>*</brp:melding>
        </brp:melding>
    </brp:meldingen>
    <brp:wijzigingPartnergegevensHuwelijk brp:objecttype="AdministratieveHandeling">
        <brp:partijCode>507013</brp:partijCode>
        <brp:tijdstipRegistratie>*</brp:tijdstipRegistratie>
    </brp:wijzigingPartnergegevensHuwelijk>
</brp:bhg_hgpActualiseerHuwelijkGeregistreerdPartnerschap_R>
